COPYRIGHT.md

```diff
@@ -0,0 +1 @@
+Copyright (c) 2005-2018, Zend Technologies USA, Inc. All rights reserved.
```

Comment:
@weierophinney Do we need to add here also link to www.zend.com?

Current copyright tag description is:
`Copyright (c) 2005-2017 Zend Technologies USA Inc. (https://www.zend.com)`.

Reply:
@weierophinney It's updated now.